distribution/docs/src/main/jdocs/jbake.properties

```diff
@@ -0,0 +1,11 @@
+site.host=http://jbake.org
```

Comment:
probably. It ultimately doesn't matter as the jbake process doesn't produce a viable site to host, just an asciidoc page to be rendered. If it'd let me remove it altogether, I would.